client/components/google-my-business/stats-nudge.js

```diff
@@ -61,11 +62,18 @@ class GoogleMyBusinessStatsNudge extends Component {
 							) }
 						</h2>
 						<div className="google-my-business__stats-nudge-button-row">
-							<Button primary onClick={ this.props.trackNudgeStartNowClicked }>
+							<Button
+								href={ `/google-my-business/${ this.props.siteSlug }` }
+								primary
+								onClick={ this.props.trackNudgeStartNowClicked }
+							>
 								{ this.props.translate( 'Start Now' ) }
 							</Button>
-							<Button onClick={ this.props.trackNudgeAlreadyListedClicked }>
-								{ this.props.translate( "I've Already Listed" ) }
+							<Button
+								href={ `/google-my-business/${ this.props.siteSlug }` }
```

Comment:
@stephanethomas We don't have a way to hide the nudge forever yet, let's handle that in a follow up PR